.gitignore

```diff
@@ -23,3 +23,5 @@ scratch
 
 # JSDoc output
 doc/api/
+
+.idea
```

Comment:
For specific IDEs, editors, etc, perhaps better to [set up the ignore on that particular machine](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)? But no big deal.